### ISSUE TITLE
Additional programmatic "Sun Event" control ?

### ISSUE THREAD

**Issue by matt6575** [mrgadget/node-red-contrib-eztimer#48]
I really appreciate the addition you made for controlling "ontype" and "offtype" and they work well.

I recently started tweaking my project again adding more user flexibility to it and assumed that if I programmatically set "on/off type" to "Sun Event" that I could then possibly use msg.payload.ontime or offtime to then set the type of Sun Event, like change Dawn to Sunrise or any other option that is available from within the node already. Obviously that appears to not be the case. Maybe there could be msg.payload.onsunevent and offsunevent properties?

**Comment by shimbone** [comment]
Yes, sun calculated events cannot be programmatically set.  So if I send payload.ontype=1and payload.ontime="dusk" the node will not set the calculated dusk time as the on time for the node.  Nor does it show any obvious errors.  It seems to just ignore ontime="dusk" and instead use whatever the last setting for a sun calculated on-time was in the UI, even if the node has since been changed to manual in the UI.

Note that in experimenting around I found that sending payload.ontime="dusk" to schedex **does** actually set the node's on time to the calculated time of dusk.

I've attached a flow that illustrates this issue.  Thanks!

[eztimer Test Flow.zip](https://github.com/mrgadget/node-red-contrib-eztimer/files/10253154/eztimer.Test.Flow.zip)
